improved translation support, UX improvments

Batcastle opened the pull request.

Batcastle committed:
update CYGO info

 * remove CYGO verification for Drauger OS users
 * Update wording on CYGO button

Batcastle committed:
update CYGO logo

Batcastle committed:
minor typo fixes

Batcastle committed:
remove all CYGO stuffs

new stuff coming in it's place

Batcastle committed:
improved language handling in the backend

 * optimize LANG environment variable parsing
 * fix bug reading language conf files
 * add support for future language JSON files

Batcastle committed:
remove CYGO link, replace with language configuration

Batcastle committed:
many UX improvements

more to come

Batcastle committed:
fix bug with "open" not being translated

Batcastle committed:
many more UX improvments

Also added link to myDrauger

Batcastle committed:
fix bugs in build script

removed references to verifier.cxx because it has been removed

Batcastle committed:
minor bug fix

fixed alignment of label on main welcome screen

pep8speaks commented:
Hello @Batcastle! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:

* In the file [`usr/share/drauger-welcome/main_ui.py`](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py):

> [Line 29:1](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L29): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 30:1](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L30): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 147:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L147): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (118 > 79 characters)
> [Line 148:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L148): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (116 > 79 characters)
> [Line 149:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L149): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (231 > 79 characters)
> [Line 150:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L150): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (357 > 79 characters)
> [Line 153:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L153): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (92 > 79 characters)
> [Line 167:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L167): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (100 > 79 characters)
> [Line 168:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L168): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (196 > 79 characters)
> [Line 169:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L169): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (131 > 79 characters)
> [Line 170:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L170): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (131 > 79 characters)
> [Line 180:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L180): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (110 > 79 characters)
> [Line 181:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L181): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (445 > 79 characters)
> [Line 182:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L182): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (173 > 79 characters)
> [Line 183:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L183): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (142 > 79 characters)
> [Line 184:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L184): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (237 > 79 characters)
> [Line 185:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L185): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (208 > 79 characters)
> [Line 186:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L186): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (251 > 79 characters)
> [Line 211:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L211): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (94 > 79 characters)
> [Line 319:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L319): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (86 > 79 characters)
> [Line 492:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L492): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (82 > 79 characters)
> [Line 529:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L529): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (93 > 79 characters)
> [Line 546:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L546): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (101 > 79 characters)
> [Line 550:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L550): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (101 > 79 characters)
> [Line 563:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L563): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (126 > 79 characters)
> [Line 614:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L614): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (85 > 79 characters)
> [Line 767:80](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L767): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (108 > 79 characters)
> [Line 892:5](https://github.com/drauger-os-development/drauger-welcome/blob/7e490a7ba69d43652a533ae6f4bd5049dfa9190f/usr/share/drauger-welcome/main_ui.py#L892): [E722](https://duckduckgo.com/?q=pep8%20E722) do not use bare 'except'



##### Comment last updated at 2020-07-11 01:30:05 UTC

Batcastle committed:
improved PEP8 conformance

Batcastle committed:
more PEP8 conformance

Batcastle committed:
final minor PEP8 conformance improvments

Batcastle committed:
specify exception

humbletyrant committed:
Commit initial changes made on vm, will fix errors

humbletyrant committed:
Final touches

Uncommented line 210, finished editing that borked on the VM, improved some wording and grammar, added some bold text for emphasis.

Batcastle committed:
minor typo fix

Batcastle committed:
add email

Batcastle committed:
margin fixes

Batcastle committed:
spacing fixes on language menu
